src/Humanizer/DateTimeHumanizeStrategy/DefaultDateTimeHumanizeStrategy.cs

```diff
@@ -8,7 +8,7 @@ namespace Humanizer.DateTimeHumanizeStrategy
     /// <summary>
     /// The default 'distance of time' -> words calculator.
     /// </summary>
-    public class DefaultDateTimeHumanizeStrategy : IDateTimeHumanizeStrategy
+    public class DefaultDateTimeHumanizeStrategy : IDateTimeHumanizeStrategy, IDateTimeOffsetHumanizeStrategy
```

Comment:
I don't think this is a good idea. I think we need to have separate strategy classes for `DateTimeOffset`. I appreciate that the implementation is more or less the same and perhaps you can just delegate the responsibility to the other class or even better extract a class for the algorithm and use it in both implementations.